README.md

```diff
@@ -11,6 +11,9 @@ duplicated git dependencies.
 
 ## Local chain usage
 
+First make sure you have [`compactc`](https://docs.midnight.network/relnotes/compact) set up and the `COMPACT_HOME` env var set to the `compactc` directory.
```

Comment:
you don't really need compact to use the batcher I think, why is this needed here?

Reply:
I'll roll back that commit and just mention it in `pvp-arena`